components/eam/cime_config/config_component.xml

```diff
@@ -82,11 +82,11 @@
       <value compset="_EAM%ADIAB"     >-phys adiabatic</value>
 
       <!-- SCREAM configurations -->
-      <value compset="_EAM%SCREAM-LR">-phys default -shoc_sgs -microphys p3 -chem none -nlev 72 -rad rrtmgp -bc_dep_to_snow_updates</value>
-      <value compset="_EAM%SCREAM-HR">-phys default -shoc_sgs -microphys p3 -chem none -nlev 128 -rad rrtmgp -bc_dep_to_snow_updates</value>
+      <value compset="_EAM%SCREAM-LR">-phys default -shoc_sgs -microphys p3 -chem spa -nlev 72 -rad rrtmgp -bc_dep_to_snow_updates</value>
```

Comment:
So simple! Quick question though: will running with spa chemistry automatically figure out which input data files to run spa with? And are the data files in the inpudata repository on ALCF?

Reply:
Okay, confirmed that the input data file is on ALCF: 
`/lcrc/group/acme/public_html/inputdata/atm/cam/chem/spa/spa_mixing_ratio_data.nc`